config/initializers/colours.rb

```diff
@@ -12,8 +12,8 @@ module Colours
     palette: {
       blue: { # blue_pale etc
         pale: '#f2f6fc'.freeze,
-        light: '#cbe4fc'.freeze, # Blue medium from original design - used in nav. Blue light in original design is not distinct enough from this, so removed
-        medium: '#83a4df'.freeze, # new colour - derived (half way colour between light and dark) using https://www.dannybrien.com/middle/
+        light: '#dcecfd'.freeze, # did not use blue medium from original design as it is not distinct enough from light
```

Comment:
Note the very slight change in the blue light (is now actually blue light from the original design, it was the blue medium before). This is the one I showed Claudia in the screen grabs as I think it looks marginally better in large blocks next to the other colours than the other one but in smaller detail, you'd not notice the change all. Have changed the derived new medium slightly too as it's the halfway point between blue light and bright.